mlir/include/mlir/IR/BuiltinTypes.td

```diff
@@ -838,6 +838,26 @@ def Builtin_MemRef : Builtin_Type<"MemRef", "memref", [
     ///
     bool areTrailingDimsContiguous(int64_t n);
 
+    /// Return the maximum number of trailing dimensions that are
```

Comment:
```suggestion
    /// Return the number of trailing dimensions that are
```

Maybe 'maximum' is redundant, and the method could be called `getNumContiguousTrailingDims()`

Reply:
Sounds good.